prow/getting_started_deploy.md

```diff
@@ -199,18 +201,24 @@ $ bazel run //experiment/update-hook -- \
   --confirm=false  # Remove =false to actually add hook
 ```
 
-If you don't want to use the `add-hook` utility, go to your org or repo and click `Settings -> Webhooks`.
-
 Look for the `http://an.ip.addr.ess/hook` you added above.
 A green check mark (for a ping event, if you click edit and view the details of the event) suggests everything is working!
 
-You can click `Add webhook` on the Webhooks page to add the hook manually,
-if you do not want to use the `update-hook` utility:
-- Go to your org or repo and click `Settings -> Webhooks`, and click `Add webhook`
+2. If you do not want to use the `update-hook` utility, you can go the GitHub web page and add the hook manually:
+
+- Go to your org or repo and click `Settings -> Webhooks`, and click `Add webhook`.
 - Change the `Payload URL` to `http://an.ip.addr.ess/hook` you are planning to add.
 - Change the `Content type` to `application/json`, and change your `Secret` to the `hmac-path` secret you created above.
-- Change the trigger to `Send me **everything**.`
-- Click `Add webhook` 
+- Change the trigger to `Send me **everything**`.
+- Click `Add webhook`.
+
+### Use `hmac` tool to manage webhooks and hmac tokens (recommended)
+
+If you need to configure webhooks for multiple orgs or repos, the manual process does not work that well as it
+can be error-prone, and it'll be painful when you want to replace the hmac token if it is accidentally leaked.
+
+In such case, it's recommended to use the [hmac][9] tool to automatically manage the webhooks
```

Comment:
This is easier:

```
it's recommended to use the [hmac] tool to automatically...


[hmac]: /prow/cmd/hmac/README.md
```

No need for the pointless number IMO

Reply:
We already have [1]-[8] in https://github.com/kubernetes/test-infra/pull/18064/files#diff-11cf136e2fed06b6cb359e3025187497R601-R608 so I feel it's probably better to keep them consistent?

Reply:
It is pretty confusing that [9] comes before [6]. What is the point? Why is [9] valuable to the person reading this?

Reply:
IMO you should either sort these so that [6] comes before [9] or better yet drop this physical book pattern in favor of something more appropriate for a digital document.

Reply:
I see, I actually don't need this so I changed it to be just `[hmac](/prow/cmd/hmac/README.md)`..